README.md

```diff
@@ -40,4 +40,19 @@ Returns `Promise<Module[]>`
 
 Will walk your entire node_modules tree reporting back an array of "modules", each
 module has a "path", "name" and "depType".  See the typescript definition file
-for more information.
```

Comment:
oops didn't mean to rid this, fixing

Reply:
I cannot tell why they are still considered different, spacing looks identical and line endings haven't been changed afaict. Ignoring since it's effectively the same